pkg/apis/pravega/v1alpha1/pravegacluster_types.go

```diff
@@ -196,6 +205,23 @@ func (tp *TLSPolicy) IsSecureSegmentStore() bool {
 	return len(tp.Static.SegmentStoreSecret) != 0
 }
 
+type AuthenticationParameters struct {
+	// Enabled specifies whether or not authentication is enabled
+	// By default, authentication is not enabled
+	Enabled bool `json:"enabled"`
+
+	// name of Secret containing Password based Authentication Parameters like username, password and acl
+	// optional - used only by PasswordAuthHandler for authentication
+	PasswordAuthSecret string `json:"passwordAuthSecret,omitempty"`
```

Comment:
Next time, please consider the conventions of the Kubernetes API.   References to secrets are typically named "secretRef".  I would also have considered introducing a `password` sub-section, because a) the password handler is merely one of numerous possible implementations, and b) could have other configurable elements in future.   In sum I would have suggested:
```
spec:
  authentication:
    password: 
      secretRef: my-password-file

```